### DISCUSSION TITLE
Invalid File Size

### DISCUSSION THREAD

**Discussion by lockevod** [devemux86/cruiser#464] [Q&A] [answered]
Hello! I have a map (generated by me with osmosis, etc). This map has a size of 33 MB and when I try to open in cruiser (last version) i have an error (invalid size map). I suppose the problem is the map is bigger than the max ... but i don't know howto resolve and what is max size.

Thanks for your help

**Comment by lockevod** [comment]
I've reduced the size.. (28M so-so) but the error message is the same Invalid size: 33395188 .... it's strange because this isn't the current size (I've doubled checked and I'm opening the correct file).

**Comment by devemux86** [comment, marked as answer]
Hi @lockevod,

Have you created an offline map with [Mapsforge map-writer](https://github.com/mapsforge/mapsforge/blob/master/docs/MapCreation.md)?

This warning happens when the file size in map header is different from the actual file size.

So the map file is not valid. Probably was not created correctly or cannot be read properly.

**Reply by lockevod** [reply]
thanks it's strange because it's a copy the same file and rename this copy.. i can open it .. but if I try to open the original.. is there any cache in cruiser or something like this?

thanks for your help

**Reply by devemux86** [reply]
Android or desktop Cruiser?

Or are you trying to open the file from the cloud?
